google-cloud-storage/src/test/java/com/google/cloud/storage/V4SigningTest.java

```diff
@@ -104,14 +105,14 @@ public void test() {
 
     SignUrlOption style = SignUrlOption.withPathStyle();
 
-    if (testData.getUrlStyle().equals(SigningV4Test.UrlStyle.VIRTUAL_HOSTED_STYLE)) {
+    if (testData.getUrlStyle().equals(UrlStyle.VIRTUAL_HOSTED_STYLE)) {
```

Comment:
can you split out this into a separte PR if possible?

Reply:
For this specifically, I mean to split out conformance test changes into a separate PR.